src/_posts/databases/postgresql/2000-01-01-troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Troubleshooting Scalingo for PostgreSQL
+nav: Troubleshooting
+modified_at: 2023-12-01 00:00:00
+tags: databases postgresql addon
+index: 9
+---
+
+## Identifying Performances Issues
+
+
+## Understanding Idle Queries
```

Comment:
It seems that you have inverted the titles of the 2 paragraphs. This paragraph seems to discuss memory consumption and not idle connections

Reply:
Fixed in https://github.com/Scalingo/documentation/pull/2389/commits/d9c945ae6290165b02d1de051cc66238e6da8d11